.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+concurrency:
+  group: ci-${{ github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+env:
+  TARGET_BRANCH: ${{ github.base_ref || github.ref }}
+
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+      - name: Lint chart
+        run: ct lint --target-branch ${TARGET_BRANCH##*/} --lint-conf lintconf.yaml --config ct.yaml --chart-yaml-schema chart_schema.yaml
```

Comment:
When combined with the above we could use `${TARGET_BRANCH}` I think? My concern with this substitution is that if at some point in the future we adopt branch names like `release/X.Y` then this will break.

Reply:
Right that makes sense! I updated this to get the short branch name directly.